src/ethereum_spec_tools/evm_tools/t8n/__init__.py

```diff
@@ -352,6 +352,8 @@ def apply_body(self) -> None:
                     receipt,
                 )
 
+                self.txs.add_receipt(tx)
```

Comment:
You can add `gas_used` as an additional parameter which you can access in the `apply_body` function as `process_transaction_return[0]`